llvm/test/MC/AMDGPU/gfx12_asm_ds_alias.s

```diff
@@ -27,5 +27,11 @@ ds_min_rtn_f64 v[5:6], v1, v[2:3]
 ds_subrev_u32 v1, v2
 // GFX12: ds_rsub_u32 v1, v2                      ; encoding: [0x00,0x00,0x08,0xd8,0x01,0x02,0x00,0x00]
 
+ds_subrev_rtn_u32 v5, v1, v2
```

Comment:
do we need tests that the alias is rejected on other targets?

Reply:
I don't think we normally go that far, but I suppose it can't hurt to check that they are not recognized on GFX11.

Reply:
There is nothing inherently GFX12-specific about these aliases. It is just that SP3 happened to add them in the GFX12 timeframe.